### DISCUSSION TITLE
How do I set up package label updates?

### DISCUSSION THREAD

**Discussion by qcyblm** [renovatebot/renovate#11017] [Request Help]
https://github.com/qcyblm/qcyblm.github.io/commit/b8164e450037a958eb453e2554c1dc43e05ec42d 
"@vuepress/plugin-last-updated" next to laster,I just want the latest package next label，not the latest package latest label.

**Comment by viceice** [comment]
https://docs.renovatebot.com/configuration-options/#followtag
